etl/steps/data/garden/demography/2023-11-08/modal_age_death.meta.yml

```diff
@@ -0,0 +1,37 @@
+# NOTE: To learn more about the fields, hover over their names.
+definitions:
+  common:
+    presentation:
+      topic_tags:
+        - Life Expectancy
+
+
+# Learn more about the available fields:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/dataset/
+dataset:
+  update_period_days: 365
+
+
+# Learn more about the available fields:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/tables/
+tables:
+  modal_age_death:
+    # Learn more about the available fields:
+    # http://docs.owid.io/projects/etl/architecture/metadata/reference/indicator/
+    variables:
+      modal_age_death:
+        title: Adult modal age at death
+        unit: years
+        description_short: |-
+          The most common age at death, among adults.
+
+        description_key:
+          - The adult modal age at death is the age (beyond infancy) at which the majority of deaths occur.
+          - In a recent paper by Diaconu et al. (2022), the authors propose the modal age at death for measuring occupational inequalities at older ages in Finland. They compare trends in modal age at death with those of more conventional measures of mortality at older ages, such as life expectancy at age 65, 75.
+          - |-
+            Compared to life expectancy at birth or at age 65, M has the advantages of being: (i) determined solely by mortality at older ages, (ii) free from an arbitrary choice of "old" age threshold, and (iii) compares individuals with similar survival chances over time.
```

Comment:
Maybe worth adding what 'M' means here, I think it's clarified in the dataset description but could be worth adding again.